relm4/src/component/shutdown_on_drop.rs

```diff
@@ -3,29 +3,29 @@ use tokio::sync::oneshot;
 /// A type that destroys an [`AsyncComponent`](crate::async_component::AsyncComponent)
 /// as soon as it is dropped.
 #[derive(Debug)]
-pub(crate) struct DestroyOnDrop {
+pub(crate) struct ShutdownOnDrop {
```

Comment:
I'd move this to `component/mod.rs` or at least re-export it there to avoid needing to use this module name elsewhere.